src/api/GoalsAPI/index.ts

```diff
@@ -353,14 +353,27 @@ export const addHintGoaltoMyGoals = async (goal: GoalItem) => {
   ]);
 };
 
-export const fetchArchivedGoalByTitle = async (value: string) => {
-  if (value.trim() === "") {
+export const fetchArchivedGoalByTitle = async (goalId: string, parentGoalId: string) => {
```

Comment:
if we say byTitle then arg should be named as title. can we change that goalId to goalTitle or title?